submissions/burning-questions/burning-questions.md

```diff
@@ -0,0 +1,382 @@
+
+`Percent gain = 84.6834 MJ / 40 MJ = 211.7085% ≈ 212%`
+
+Compared to no modules offering 103% gain, this setup offers an excellent gain of 212%, but the full-productivity setup with 233% gain is better. This is because the cumulative fuel gains from the third productivity 3 module outweigh the reduced power consumption from the speed 3 module. 
+
+Instead of replacing the third productivity module with a speed module, we can introduce speed beacons. This would allow us to both keep the maximum productivity and cut down machine power consumptions, although the beacons themselves would introduce additional power consumptions to account for. I estimate that good boosts in energy gains are possible by having each beacon affect multiple machines. However, since calculations involving beacons would include additional factors to consider such as machine counts and the physical layout of the fuel processing area, I prefer to leave that rabbit hole alone.
+
+Meanwhile, seeing the strong effects of productivity modules, I wonder if we can use them to make rocket fuel production yield net energy gains. So, let's fill the assembling machine 2 with productivity 3 modules:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing solid fuel into rocket fuel in an assembling machine 2 at maximum productivity'
+ caption='Processing solid fuel into rocket fuel in an assembling machine 2 at maximum productivity'
+
+%}
+
+Well, the input energy cost and output energy generation are equal now, but the processing energy cost has become too high, having nearly quadrupled, making the overall process still cost more energy than it gains. 
+
+However, we have other options for making rocket fuel: We can use assembling machine 3s instead and take advantage of their four module slots. Here is how some different combinations perform:
+
+{% include image.html
+
+src= 'src'
+ alt='Processing solid fuel into rocket fuel in an assembling machine 3 with various module combinations'
+ caption='Processing solid fuel into rocket fuel in an assembling machine 3 with various module combinations''
+
+%}
+
+So, I tried four productivity 3 modules in an assembling machine 3 but this slowed down the machine so much that the processing energy was almost more than the total fuel energy it put out! Next, I tried to dampen some of the inflated power consumption and speed loss by replacing the fourth productivity 3 module with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed strategy and used a speed module 3 instead of the efficiency 3 module, to restore the speed of the machine and cut down the processing time. While this saved more energy than the efficiency module, it still did not make enough of a difference. I tried more module combinations after that and even tried adding various beacon combinations too (no diagrams for those) but no matter what I only managed to get close to a breakeven. 
```

Comment:
```suggestion
So, I tried four productivity 3 modules in an assembling machine 3, but this slowed down the machine so much that the processing energy was almost more than the total fuel energy it put out! Next, I tried to dampen some of the inflated power consumption and speed loss by replacing the fourth productivity 3 module with an efficiency 3 module. This was helpful but insufficient. Afterwards I changed strategy and used a speed module 3 instead of the efficiency 3 module, to restore the speed of the machine and cut down the processing time. While this saved more energy than the efficiency module, it still did not make enough of a difference. I tried more module combinations after that and even tried adding various beacon combinations too (no diagrams for those) but no matter what I only managed to get close to a breakeven. 
```